Source/Core/PixelFormat.js

```diff
@@ -150,6 +150,7 @@ define([
                 // Many GPUs store RGB as RGBA internally
                 // https://devtalk.nvidia.com/default/topic/699479/general-graphics-programming/rgb-auto-converted-to-rgba/post/4142379/#4142379
```

Comment:
Since we return 3 now, should this comment be removed?

Reply:
I'm surprised having it as 4 didn't cause any problems since we use this to create arrays. I guess we don't use RGB textures at all. I kept this there for now because I plan on moving it to a separate function (for when we measure GPU memory usage)